lib/matplotlib/text.py

```diff
@@ -541,51 +541,15 @@ def _update_clip_properties(self):
         if self._bbox_patch:
             bbox = self._bbox_patch.update(clipprops)
 
-    def set_clip_box(self, clipbox):
-        """
-        Set the artist's clip `~.transforms.Bbox`.
-
-        Parameters
-        ----------
-        clipbox : `matplotlib.transforms.Bbox`
-        """
+    def set_clip_box(self, clipbox):  # inherited docstring.
```

Comment:
It's good to have a comment for the inherited docstring. This documents the intention not to have a docstring there and prevents future accidential additions. Can we move the comment
to the place where the docstring should usually be:

~~~
def set_clip_box(self, clipbox):
    # docstring inherited
~~~

That way it's easy to see and we don't run into linewidth limitations with longer signatures.

Also, for some reason, I'd prefer `# docstring inherited` over `# inherited docstring` if you don't mind. (Can't really explain, but it feels more appropriate.)

Reply:
done